core/src/main/java/com/linecorp/armeria/common/RequestContext.java

```diff
@@ -61,7 +62,7 @@
  * A server-side {@link Request} has a {@link ServiceRequestContext} and
  * a client-side {@link Request} has a {@link ClientRequestContext}.
  */
-public interface RequestContext {
+public interface RequestContext extends Unwrappable {
```

Comment:
Should we override the return type of `unwrap()`? `ClientRequestContext` and `ServiceRequestContext` also need to override it.

Reply:
Good point 👍  Done!